Formula/binwalk.rb

```diff
@@ -67,13 +74,13 @@ class Binwalk < Formula
   end
 
   resource "python-dateutil" do
-    url "https://files.pythonhosted.org/packages/be/ed/5bbc91f03fa4c839c4c7360375da77f9659af5f7086b7a7bdda65771c8e0/python-dateutil-2.8.1.tar.gz"
-    sha256 "73ebfe9dbf22e832286dafa60473e4cd239f8592f699aa5adaf10050e6e1823c"
+    url "https://files.pythonhosted.org/packages/4c/c4/13b4776ea2d76c115c1d1b84579f3764ee6d57204f6be27119f13a61d0a9/python-dateutil-2.8.2.tar.gz"
+    sha256 "0123cacc1627ae19ddf3c27a5de5bd67ee4586fbdd6440d9748f8abb483d3e86"
   end
 
-  resource "six" do
-    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
-    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+  resource "tk" do
```

Comment:
I think the upstream listed wrong `tk`. Though, not even sure either is still used:
- PyPI `tk` is TensorKit, which I doubt is used here.
- Apt package `python3-tk` is for Tkinter. I don't see it directly used, so not sure if it is still needed. It is an optional dependency for `matplotlib`.

https://github.com/ReFirmLabs/binwalk/commit/afe921fcbecf4dcc56c57254501ded1809d3bb98#diff-0ee353a10a4e0fc85c5abd65ceb6000cb3471159998bb41ece6a0ba87ba116f2 changed `python3-tk` (Tkinter) to `tk` (TensorKit)

It looks like some `matplotlib` API usage (at least previously) has required Tkinter - e.g. https://github.com/ReFirmLabs/binwalk/issues/304

Reply:
Tkinter is in python core, so we should be good in there, I am gonna PR to remove tk dependency (as it is probably a wrong dependency)